src/AboutDialog.cpp

```diff
@@ -141,11 +141,20 @@ void AboutDialog::CreateCreditsList()
    // members. Also considered, add an extra parameter to AddCredit,
    // indicating if they were an Audacity developer.
 
-   // The Saucedacity and Tenacity Teams
-   AddCredit(wxT("Avery King"), saucedacityFounderFormat, roleSaucedacityTeamMember);
-   AddCredit(wxT("Panagiotis \"AlwaysLivid\" Vasilopoulos ([[https://alwayslivid.com|Website]])"), tenacity_contributorFormat, roleTenacityTeamMember);
-   AddCredit(wxT("Rikard \"akleja\" Jansson ([[https://github.com/akleja|GitHub]])"), tenacity_contributorFormat, roleTenacityTeamMember);
-   AddCredit(wxT("Mart \"leio\" Raudsepp ([[https://github.com/leio|GitHub]])"), tenacity_contributorFormat, roleTenacityTeamMember);
+   // The Tenacity Team
+   AddCredit("Avery \"Generic Person\" King ([[https://github.com/generic-pers0n|GitHub]])", tenacity_leadDeveloperFormat, roleTeamMember);
+   AddCredit("Rikard \"akleja\" Jansson ([[https://github.com/akleja|GitHub]])", tenacity_contributorFormat, roleTeamMember);
+   AddCredit("Mart \"leio\" Raudsepp ([[https://github.com/leio|GitHub]])", tenacity_contributorFormat, roleTeamMember);
+
```

Comment:
Some additional credits that I did not update myself would be nice -- I already addressed this in private.